unreleased_history/performance_improvements/parallel_notify-one_i_LaunchParallelMemTableWriters.md

```diff
@@ -0,0 +1,18 @@
+# Improved write throughput to memtable when there's a large number of concurrent writers and allow_concurrent_memtable_write=true(#12545)
```

Comment:
nit: # is not needed. We use unreleased_history/release.sh to generate release note which will just prepend `*` for this changelog: https://github.com/facebook/rocksdb/blob/c72ee4531b288bf08b9414155fafb86cc4378fb4/unreleased_history/README.txt#L25-L28

Reply:
# has been removed.